package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@fxts/core",
   "version": "0.12.0",
   "description": "A functional library for TypeScript/JavaScript programmers.",
-  "type": "module",
```

Comment:
This value is required. Otherwise, an error occurs when the user uses the `type: module`

Reply:
Thank you for your review.
I modified it to work by adding `package.json` to the `esm` directory.

You can check it out in this repository: https://github.com/shine1594/fxts-test